### PR TITLE
DEV-4337 docs(string_tokenizer_count): add description for exercise

### PR DESCRIPTION
**Pull request by eslopfer**
Add description for exercise.

### CODE REVIEW

File: subjects/devops/string_tokenizer_count/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+## string_tokenizer_count
+
+### Instructions
+
+Create a file string_tokenizer_count.py that contains a function tokenizer_counter which takes in a string as a parameter and returns a dictionary of words and their count in the string.
```
Comment (nprimo) [inline subjects/devops/string_tokenizer_count/README.md:5]: I would add some formatting to better highlight the name of the expected file and expected function. For example

`string_tokenizer_count.py` and `tokenizer_counter`

File: subjects/devops/string_tokenizer_count/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```python
+string = "This is a test sentence, with various words and 123 numbers!"
+result = tokenizer_counter(string)
+print(string)
+```
+
+And its output:
+
+```console
```
Comment (nprimo) [inline subjects/devops/string_tokenizer_count/README.md:23]: You forgot to quote the result of the script here :)

File: subjects/devops/string_tokenizer_count/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- The function should return a dictionary of words and their count, sorted alphabetically by word.
+
+### Usage
+
+Here is an example of how to use the function:
+
+```python
+string = "This is a test sentence, with various words and 123 numbers!"
+result = tokenizer_counter(string)
+print(string)
```
Comment (nprimo) [inline subjects/devops/string_tokenizer_count/README.md:18]: this should be `print(result)`

File: subjects/devops/string_tokenizer_count/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+```console
+string = "This is a test sentence, with various words and 123 numbers!"
+result = tokenizer_counter(string)
+```
+
+### Hints
+
+- The `re` module can be used to remove non-alphanumeric characters.
+
+- The `collections` module can be used to count the words.
```
Comment (nprimo) [inline subjects/devops/string_tokenizer_count/README.md:32]: I would be more explicit and add they can use the `Counter` class of this module to be in line with the other subjects

File: subjects/devops/string_tokenizer_count/README.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+Create a file string_tokenizer_count.py that contains a function tokenizer_counter which takes in a string as a parameter and returns a dictionary of words and their count in the string.
+
+- The function should remove any punctuation from the string and convert it to lowercase before counting the words.
+
+- The function should return a dictionary of words and their count, sorted alphabetically by word.
+
+### Usage
+
+Here is an example of how to use the function:
```
Comment (nprimo) [inline subjects/devops/string_tokenizer_count/README.md:13]: In other subjects, the below approach has been used to cover this part.

Present the python script (e.g. `tests.py`) content and then, reporting the result of the console as follows:
```consoles
$ python3 test.py
<expected output>
$
```

**Review by nprimo — CHANGES_REQUESTED** [review]
I left some comments inline

File: subjects/devops/string_tokenizer_count/README.md (hunk 1 above)
Comment (eslopfer) [reply subjects/devops/string_tokenizer_count/README.md:5]: Thanks. Done.

File: subjects/devops/string_tokenizer_count/README.md (hunk 3 above)
Comment (eslopfer) [reply subjects/devops/string_tokenizer_count/README.md:18]: Thanks, fixed.

File: subjects/devops/string_tokenizer_count/README.md (hunk 4 above)
Comment (eslopfer) [reply subjects/devops/string_tokenizer_count/README.md:32]: Thanks. Done.

File: subjects/devops/string_tokenizer_count/README.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+result = tokenizer_counter(string)
+```
+
+### Hints
+
+- The `re` module can be used to remove non-alphanumeric characters.
+
+- The `collections` module can be used to count the words.
+
+- The `operator` module can be used to sort the dictionary alphabetically by word.
```
[the remark this answers is not in the document]
Comment (eslopfer) [reply subjects/devops/string_tokenizer_count/README.md:34]: Thanks. Done.

File: subjects/devops/string_tokenizer_count/README.md (hunk 5 above)
Comment (eslopfer) [reply subjects/devops/string_tokenizer_count/README.md:13]: Thanks, done.

File: subjects/devops/string_tokenizer_count/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+### Hints
+
+- The `re` module can be used to remove non-alphanumeric characters.
+
+- The `Counter` class of the `collections` module can be used to count the words.
+
+- The `operator` module can be used to sort the dictionary alphabetically by word.
+
+Here is an example of how to sort a dictionary in python, using a test.py script:
```
Comment (nprimo) [inline subjects/devops/string_tokenizer_count/README.md:37]: **a suggestion** you can highlight the `test.py` name to make it stand out

**Review by nprimo — APPROVED** [review]
LGTM :rocket: 

I left a small suggestion: I will let you chose if you want to implement in or not

File: subjects/devops/string_tokenizer_count/README.md (hunk 7 above)
Comment (eslopfer) [reply subjects/devops/string_tokenizer_count/README.md]: Thanks, I did it :+1:

**Pull request merged**